Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -1724,6 +1724,22 @@ The mini form task is used to test survey forms functionality (`ORKFormStep`).
     }
     
     {
+        /*
+         A text question with single-line text entry, text validation, and a URL keyboard.
+         */
+        ORKTextAnswerFormat *format = [ORKAnswerFormat textAnswerFormatWithValidationExpression:@"^(https?:\\/\\/)?([\\da-z\\.-]+)\\.([a-z\\.]{2,6})([\\/\\w \\.-]*)*\\/?$" invalidMessage:@"Enter a valid URL."];
+        format.multipleLines = NO;
+        format.keyboardType = UIKeyboardTypeURL;
+        format.autocapitalizationType = UITextAutocapitalizationTypeNone;
+        format.autocorrectionType = UITextAutocorrectionTypeNo;
+        format.spellCheckingType = UITextSpellCheckingTypeNo;
+        ORKQuestionStep *step = [ORKQuestionStep questionStepWithIdentifier:@"qid_005c"
+                                                                      title:@"What is your website?"
+                                                                     answer:format];
+        [steps addObject:step];
```

Comment:
Can we just add this into the MiniForm form step (where the email is at the moment also)?
We already have the example of it being used in a survey in `ORKCatalog`.